### ISSUE TITLE
Mining should not start until initial full sync is complete.

### ISSUE THREAD

**Issue by arkpar** [ethcore/parity#1056]

**Comment by gavofyork** [comment]
not a priority. just slightly more manual that things need to be at present.

this used to be a feature but we removed it due to issues with knowing what constitutes a full sync.

**Comment by tomusdrw** [comment]
Related #1255 #1280  ?

**Comment by gavofyork** [comment]
done.
